### PR TITLE
fix: overflow of dialog-content-container

### PR DESCRIPTION
**Pull request by Juskinbo**
fixed the overflow of dialog-content-container
~~old~~
old

new
new

### CODE REVIEW

File: web/src/components/AskAIDialog.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ const AskAIDialog: React.FC<Props> = (props: Props) => {
           <Icon.X />
         </button>
       </div>
-      <div className="dialog-content-container !w-112">
+      <div className="dialog-content-container">
```
Comment (boojack) [inline web/src/components/AskAIDialog.tsx:82]:
```suggestion
      <div className="dialog-content-container !w-112 max-w-full">
```

File: web/src/components/Dialog/BaseDialog.tsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ const BaseDialog: React.FC<Props> = (props: Props) => {
 
   return (
     <div className={`dialog-wrapper ${className ?? ""}`} onMouseDown={handleSpaceClicked}>
-      <div ref={dialogContainerRef} className="dialog-container" onMouseDown={(e) => e.stopPropagation()}>
+      <div ref={dialogContainerRef} className="dialog-container w-112" onMouseDown={(e) => e.stopPropagation()}>
```
Comment (boojack) [inline web/src/components/Dialog/BaseDialog.tsx:59]: `BaseDialog` is a base component and should not have a special width. Add a `max-w-full` to `AskAIDialog` will fix the overflow problem.

```suggestion
      <div ref={dialogContainerRef} className="dialog-container" onMouseDown={(e) => e.stopPropagation()}>
```

**Review by boojack — CHANGES_REQUESTED** [review]

**Review by boojack — APPROVED** [review]
LGTM, thanks for your contribution!

**Review by Juskinbo — COMMENTED** [review]
fixed

**Pull request merged**